### DISCUSSION TITLE
Readability tweaks for the beatmap editor

### DISCUSSION THREAD

**Discussion by riesha** [ppy/osu#18104] [Ideas] [answered]
Beatmap readability in the lazer editor is currently much worse than stable, readability is very important to the mapping experience both while mapping and while looking at maps.

### Objects turning white after having been hit in the editor 
This is  used for pattern structuring and avoiding overlaps with previous objects and to also quickly get information on which objects have been hit and which objects are going to be hit which is important while mapping
 Here are some comparisons between lazer and stable
![image](https://user-images.githubusercontent.com/58556779/166916611-1eb01185-6b0a-457c-bc45-25b4d3d94954.png)
![image](https://user-images.githubusercontent.com/58556779/166917325-396ba725-5f88-49f1-9256-6e620fd6f7a5.png)
![image](https://user-images.githubusercontent.com/58556779/166912931-55aa72f6-120f-4eb2-a2bc-5583a4d4bce4.png)
![image](https://user-images.githubusercontent.com/58556779/166915403-1ac4df8a-73e5-4d6d-9ecc-b1ec9a6ed13c.png)

### Kiai flashing in the editor
This is a more subjective thing but the combination of fading + kiai flashing make playing a map in the editor a painful experience visually especially for people suffering from epilepsy or other conditions

https://user-images.githubusercontent.com/58556779/166917887-93900e00-530d-47f7-a16b-b2ece0f2eef0.mp4

https://user-images.githubusercontent.com/58556779/166918114-06ab7e1c-3ff2-456f-be04-22174c94d2d9.mp4

### Suggestions

- Make hit objects behave like stable (turn white when hit)
- Disable kiai flashing during editor playback

**Comment by frenzibyte** [comment, marked as answer]
I think this is actually exactly what https://github.com/ppy/osu/discussions/14298 conveys, which may indicate this to be a duplicate.

**Reply by riesha** [reply]
I apologize as i couldn't find that discussion when searching prior to posting, it does make the same points (except it doesn't mention how the kiai flashing is also bad for accessibility reasons) so i guess this can be closed or merged or whatever needs to be done I've never used discussions before
